### PR TITLE
CORE-5079: Change names/descriptions for Open API Resources

### PR DESCRIPTION
**Pull request by vkolomeyko**

### CODE REVIEW

**Comment by vkolomeyko** [comment]
I will wait for #1636 to be merged before integrating this change.

File: libs/virtual-node/cpi-upload-endpoints/src/main/kotlin/net/corda/libs/cpiupload/endpoints/v1/CpiUploadRPCOps.kt [hunk 1]
```diff
@@ -22,8 +22,8 @@ interface CpiUploadRPCOps : RpcOps {
      * Please note that this method will not close [cpiContent] input stream, the caller must close it.
      */
     @HttpRpcPOST(
-        title = "Upload a CPI",
-        description = "Uploads a CPI",
+        title = "CPI API",
```
Comment (barrylapthorn) [inline libs/virtual-node/cpi-upload-endpoints/src/main/kotlin/net/corda/libs/cpiupload/endpoints/v1/CpiUploadRPCOps.kt:25]: CPI management API perhaps?  Though happy to leave as-is.

File: libs/virtual-node/virtual-node-endpoints-maintenance/src/main/kotlin/net/corda/libs/virtualnode/maintenance/endpoints/v1/VirtualNodeMaintenanceRPCOps.kt [hunk 2]
```diff
@@ -12,8 +12,8 @@ import net.corda.libs.cpiupload.endpoints.v1.CpiUploadRPCOps
  * Some of them could be highly disruptive, so great care should be taken when using them.
  */
 @HttpRpcResource(
-    name = "VirtualNodeMaintenanceRPCOps",
-    description = "Virtual node maintenance endpoints",
+    name = "Virtual Node Maintenance API",
```
Comment (barrylapthorn) [inline libs/virtual-node/virtual-node-endpoints-maintenance/src/main/kotlin/net/corda/libs/virtualnode/maintenance/endpoints/v1/VirtualNodeMaintenanceRPCOps.kt:18]: maybe "management" to be consistent with the CPI api name?

**Review by barrylapthorn — DISMISSED** [review]
seems reasonable - couple of comments, but happy as-is.

**Review by alexey-kadyrov-r3 — DISMISSED** [review]
LGTM

File: libs/virtual-node/virtual-node-endpoints-maintenance/src/main/kotlin/net/corda/libs/virtualnode/maintenance/endpoints/v1/VirtualNodeMaintenanceRPCOps.kt (hunk 2 above)
Comment (vkolomeyko) [reply libs/virtual-node/virtual-node-endpoints-maintenance/src/main/kotlin/net/corda/libs/virtualnode/maintenance/endpoints/v1/VirtualNodeMaintenanceRPCOps.kt:18]: Maintenance sort of assumes some sort of management. I think the main point here is to be different from `VirtualNodeRPCOps`.

**Review by driessamyn — APPROVED** [review]
LGTM

**Review by charlieR3 — APPROVED** [review]
LGTM from the membership side of things

**Pull request merged**